test/Microsoft.Azure.Devices.Edge.Test/Metrics.cs

```diff
@@ -26,7 +26,10 @@ public async Task ValidateMetrics()
             CancellationToken token = this.TestToken;
             await this.Deploy(token);
 
-            var result = await this.iotHub.InvokeMethodAsync(Context.Current.DeviceId, ModuleName, new CloudToDeviceMethod("ValidateMetrics"), token);
+            // System resource metrics take 1 minute to start. Wait before testing
+            await Task.Delay(TimeSpan.FromMinutes(1.1));
+
+            var result = await this.iotHub.InvokeMethodAsync(Context.Current.DeviceId, ModuleName, new CloudToDeviceMethod("ValidateMetrics"), CancellationToken.None);
```

Comment:
You went back to using `CancellationToken.None`. Should we continue to use `token` instead?

Reply:
yes, don't know how that went back